.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 15 * * 2"
```

Comment:
can we add a comment saying ina a human-readable way when it is?

Reply:
Done in 16b45b0bce974a491205387f7b7e57c1f5d15947.